msal/src/authority/java/com/microsoft/identity/client/AadAuthority.java

```diff
@@ -74,7 +74,7 @@
 
     @Override
     String performInstanceDiscovery(final RequestContext requestContext, final String userPrincipalName) throws MsalServiceException, MsalClientException {
-        Logger.info(TAG, requestContext, "Passed in authority " + mAuthorityUrl.toString() + " is AAD authority. "
+        Logger.infoPII(TAG, requestContext, "Passed in authority " + mAuthorityUrl.toString() + " is AAD authority. "
                 + "Start doing Instance discovery.");
```

Comment:
Could you have a look at line 67, which contains authority url, it is not a PII?